cli/src/pcluster/api/util.py

```diff
@@ -11,10 +11,13 @@
 import datetime
 import logging
 import shutil
+import subprocess
 
 import six
+from pkg_resources import packaging
 
 from pcluster.api import typing_utils
+from pcluster.constants import INCOMPATIBLE_NODEJS_VERSION_RANGE, MIN_NODEJS_VERSION
```

Comment:
What about having all Node js variable starting with the same `NODEJS` prefix?
I mean `NODEJS_INCOMPATIBLE_VERSION_RANGE` and `NODEJS_MIN_VERSION`